05_stringer/alextmz/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_IPAddrString(t *testing.T) {
```

Comment:
This should be called `TestIPAddrString` as noted above on your previous lab: https://talks.golang.org/2014/names.slide#5

Reply:
Done